app/NSObject+SaneKVO.h

```diff
@@ -0,0 +1,35 @@
+//
+//  NSObject+SaneKVO.h
+//  iSH
+//
+//  Created by Theodore Dubois on 11/10/20.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef void (^KVOBlock)(void);
+
+@interface KVOObservation : NSObject {
+    BOOL _enabled;
```

Comment:
Is this used?

Reply:
Yes, in -disable

Reply:
My actual concern is that this never seems to be set to `YES`

Reply:
There needs to be some way to mark this as disabled so it won't be disabled a second time. Also maybe someday I'll add -enable.

Reply:
My concern is that `-disable` seems to be a no-op at the moment

Reply:
Oops

Reply:
Oops